openpower/configs/p8dtu_defconfig

```diff
@@ -18,7 +18,7 @@ BR2_ROOTFS_OVERLAY="../openpower/overlay"
 BR2_ROOTFS_POST_BUILD_SCRIPT="../openpower/scripts/fixup-target-var ../openpower/scripts/firmware-whitelist"
 BR2_LINUX_KERNEL=y
 BR2_LINUX_KERNEL_CUSTOM_VERSION=y
-BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.19.26"
+BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.19.30"
```

Comment:
please squash the commits (`git commit --amend`) rather than adding another one, we want commits to be bisectable.

Reply:
I have been discussion this with Jim Yuan (SMC).  The feed back I get is they are done.  Take it or leave it @jimsmc.

@stewart-ibm do we really have to do this "squash the commit" before you allow to merge to op-build?  We spend so much time over this PR trying to do this apple polish.  I hope you are considering this.